### PR TITLE
Security overview pages

### PR DESCRIPTION
**Pull request by yanjost**

### CODE REVIEW

File: src/_posts/compliance/security-overview/2000-01-01-compliance.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+### Health Data Hosting (HDS)
+
+Scalingo is certified to host health data (HDS) in France. We have implemented the necessary security measures to
+protect the health data hosted on the platform and we are committed to protecting the privacy of the health data. See
+our HDS certification [here](/compliance/hds). You can see our Health Data Hosting Appendix
+[here](https://scalingo.com/gtc-appendix-health-data-hosting).
+
+### ISO 27001
+
+Scalingo is certified ISO 27001 and we have implemented a high level of security to protect the data hosted on the
```
Comment (SC-Jerome-S) [inline src/_posts/compliance/security-overview/2000-01-01-compliance.md:25]: Just small English typo: "ISO xxx certified" and not "certified ISO xxx"

File: src/_posts/compliance/security-overview/2000-01-01-compliance.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+platform. The ISO 27001 certification ensures that we have implemented a comprehensive information security management
+system to protect the data hosted on the platform. See our ISO 27001 certification [here](/compliance/iso-27001).
+
+### SecNumCloud
+
+Scalingo is hosting its infrastructure in the SecNumCloud certified datacenters (uniquely for the region
+`osc-secnum-fr1`). This certification ensures that the datacenters have implemented a high level of security to protect
+the data hosted on them. We are working on a SecNumCloud certification for a new range of products.
```
Comment (SC-Jerome-S) [inline src/_posts/compliance/security-overview/2000-01-01-compliance.md:33]: Should we mentioned someone the "Sovereignty" criteria ? Like "[...] _high level of security_ and guaranteeing a high level of technical, operational and legal standards. On the one hand, we offer a cloud computing service with a high level of IT hygiene, and on the other, data is protected in compliance with European law."

File: src/_posts/compliance/security-overview/2000-01-01-employee-security.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+All employees receive security training to ensure that they are aware of the security measures implemented by the
+platform and to prevent security incidents caused by human error.
+
+### Security Awareness
+
+All employees are required to follow security best practices to protect the platform from security incidents. This
+includes using strong passwords, enabling two-factor authentication, and following the security policies implemented by
+the platform.
+
+They are trained to detect phishing attempts and to report them to the security team.
```
Comment (SC-Jerome-S) [inline src/_posts/compliance/security-overview/2000-01-01-employee-security.md:20]: Be more inclusive, like "to detect phishing, social engineering, or impersonation" ?

File: src/_posts/compliance/security-overview/2000-01-01-incident-support.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+In the event of a security incident, we will communicate with our customers to keep them informed of the incident and to
+provide updates on the status of the incident. We will provide regular updates on the incident and we will inform our
+customers of any actions they need to take to protect their data. Security communication is done through our
+[status page](https://scalingostatus.com), and by chat support or email if necessary.
+
+### On-Call Rotation
+
+Our operations team is available 24/7 to respond to security incidents and to ensure the availability of the platform.
+We have an on-call rotation to ensure that there is always someone available to respond to security incidents and to
+ensure the availability of the platform. Our on-call first response time is 15 minutes.
```
Comment (SC-Jerome-S) [inline src/_posts/compliance/security-overview/2000-01-01-incident-support.md:34]: Is it true even by night ? I guess so but we must be sure if we indicate it here

File: src/_posts/compliance/security-overview/2000-01-01-operational-security.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+nav: Operations
+modified_at: 2024-06-20 00:00:00
+tags: compliance security measures
+index: 8
+---
+
+### Logical Access Control
+
+Access to the platform administration is restricted to authorized personnel only. It is secured by two-factor
+authentication and when possible uses a bastion host and a VPN connection.
```
Comment (SC-Jerome-S) [inline src/_posts/compliance/security-overview/2000-01-01-operational-security.md:12]: Maybe too much details for a public document

File: src/_posts/compliance/security-overview/2000-01-01-physical-security.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+tags: compliance security measures
+index: 1
+---
+
+### Datacenters
+
+Scalingo uses datacenters that are certified ISO 27001, ISO 27017, ISO 27018, HDS (Region `osc-secnum-fr1`) and
+SecNumCloud (Region `osc-secnum-fr1`). These certifications ensure that the datacenters have implemented a high level of
+security to protect the data hosted on them.
+
```
Comment (SC-Jerome-S) [inline src/_posts/compliance/security-overview/2000-01-01-physical-security.md:14]: For everything below in Physical section: be careful to not act anything on behalf of Outscale.
We may just want to say our Host Provider is following the best industry standards for security, and if the customer need more information, they should directly reach them or ask through our support team

**Review by SC-Jerome-S — CHANGES_REQUESTED** [review]
First review batch

File: src/_posts/compliance/security-overview/2000-01-01-application-security.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+size of the containers. This ensures that your application can handle the expected traffic volume and that it is
+protected from denial-of-service attacks or simple traffic spikes.
+
+Furthemore, if enabled, the auto-scaling feature will automatically adjust the number of containers based on, for
+example, the CPU usage of the application or the number of requests it receives.
+
+### Dependency Management
+
+We regularly update the dependencies used by the platform to ensure that they are up-to-date and that they do not
+
```
[the remark this answers is not in the document]
Comment (yanjost) [reply src/_posts/compliance/security-overview/2000-01-01-application-security.md:35]: Yes, fixing it

File: src/_posts/compliance/security-overview/2000-01-01-application-security.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+We use security headers to protect the platform from common security vulnerabilities such as cross-site scripting (XSS)
+and clickjacking. These headers are configured to prevent common attacks and to protect the platform from known security
+vulnerabilities.
+
+### Encryption
+
+All data transmitted between the platform and the user is encrypted using HTTPS. This ensures that the data is protected
+from eavesdropping and tampering. The allowed ciphers are regularly updated to ensure that the platform is protected
+from the latest security vulnerabilities.
```
[the remark this answers is not in the document]
Comment (yanjost) [reply src/_posts/compliance/security-overview/2000-01-01-application-security.md:46]: I added some details here

File: src/_posts/compliance/security-overview/2000-01-01-auditing.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+- Regular security audits to identify vulnerabilities and to ensure that our security measures are effective.
+- Regular security training for our employees to ensure that they are aware of the security measures implemented by the
+  platform and to prevent security incidents caused by human error.
+- Regular security reviews of our security policies and procedures to ensure that they are up-to-date and effective.
+- Regular reviews of our policies and procedures to ensure that they comply with industry best practices and regulatory
+  requirements.
+
+### Audits
+
+We conduct regular external security audits to ensure that our platform is secure.
```
[the remark this answers is not in the document]
Comment (yanjost) [reply src/_posts/compliance/security-overview/2000-01-01-auditing.md:22]: OK, I fixed that

File: src/_posts/compliance/security-overview/2000-01-01-backups.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+modified_at: 2024-06-20 00:00:00
+tags: compliance security measures backups
+index: 6
+---
+
+### Backup Policy
+
+We perform regular backups of the data hosted on the platform to protect it from data loss. The backups are stored in a
+secure redundant storage system to ensure that the data is protected from hardware failures and other disasters.
+
```
[the remark this answers is not in the document]
Comment (yanjost) [reply src/_posts/compliance/security-overview/2000-01-01-backups.md:13]: Better now ?

File: src/_posts/compliance/security-overview/2000-01-01-code-security.md [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+We regularly update the dependencies used by the platform to ensure that they are up-to-date and that they do not
+contain any known security vulnerabilities. We use automated tools to scan the dependencies for security vulnerabilities
+and we update them as soon as a vulnerability is detected.
+
+### Vulnerability Management
+
+We have a vulnerability management program in place to detect and respond to security vulnerabilities in the platform.
+We use automated tools to scan the platform for security vulnerabilities and we have procedures in place to patch the
+vulnerabilities as soon as they are detected.
+
```
[the remark this answers is not in the document]
Comment (yanjost) [reply src/_posts/compliance/security-overview/2000-01-01-code-security.md:41]: As it is not setup yet, I would talk about it

File: src/_posts/compliance/security-overview/2000-01-01-employee-security.md (hunk 3 above)
Comment (yanjost) [reply src/_posts/compliance/security-overview/2000-01-01-employee-security.md:20]: ack

File: src/_posts/compliance/security-overview/2000-01-01-incident-support.md (hunk 4 above)
Comment (yanjost) [reply src/_posts/compliance/security-overview/2000-01-01-incident-support.md:34]: How would you change the sentence ?

File: src/_posts/compliance/security-overview/2000-01-01-operational-security.md (hunk 5 above)
Comment (yanjost) [reply src/_posts/compliance/security-overview/2000-01-01-operational-security.md:12]: There is no actionnable information here

File: src/_posts/compliance/security-overview/2000-01-01-physical-security.md (hunk 6 above)
Comment (yanjost) [reply src/_posts/compliance/security-overview/2000-01-01-physical-security.md:14]: Are you sure ? We have this information from Outscale

File: src/_posts/compliance/security-overview/2000-01-01-code-security.md [hunk 12]
```diff
@@ -0,0 +1,48 @@
...
+All code changes are reviewed by our development team to ensure that they do not introduce security vulnerabilities. We
+use automated tools to scan the code for common security vulnerabilities and we perform manual code reviews to catch any
+vulnerabilities that the automated tools may have missed.
+
+### Secure Development
+
+All code changes are reviewed by our development team to ensure that they do not introduce security vulnerabilities. We
+use automated tools to scan the code for common security vulnerabilities.
+
```
[the remark this answers is not in the document]
Comment (yanjost) [reply src/_posts/compliance/security-overview/2000-01-01-code-security.md:29]: Not here, but in the "audit" section

File: src/_posts/compliance/2000-01-01-security-overview.md [hunk 13]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (leo-scalingo) [inline src/_posts/compliance/2000-01-01-security-overview.md:1]: To me it's a bit weird to have `Security` and `Compliance` in the navigation, then in `Compliance` having `Security Overview`, I've the feeling it's not really intuitive

Comment (yanjost) [reply src/_posts/compliance/2000-01-01-security-overview.md:1]: The reasoning was : it's always for compliance reasons that you want to have a security overview

The "security" section is very operational, whereas the overview is all about rules and theory

Would you prefer having an "overview" section under "security" ?

File: src/_posts/compliance/security-overview/2000-01-01-compliance.md [hunk 14]
```diff
@@ -0,0 +1,33 @@
...
+
+### ISO 27001
+
+Scalingo is certified ISO 27001 and we have implemented a high level of security to protect the data hosted on the
+platform. The ISO 27001 certification ensures that we have implemented a comprehensive information security management
+system to protect the data hosted on the platform. See our ISO 27001 certification [here](/compliance/iso-27001).
+
+### SecNumCloud
+
+Scalingo is hosting its infrastructure in the SecNumCloud certified datacenters (uniquely for the region
```
Comment (leo-scalingo) [inline src/_posts/compliance/security-overview/2000-01-01-compliance.md:31]:
```suggestion
Scalingo is hosting its infrastructure in the SecNumCloud qualified datacenters (uniquely for the region
```

I think that's the right word to use

File: src/_posts/compliance/security-overview/2000-01-01-compliance.md [hunk 15]
```diff
@@ -0,0 +1,33 @@
...
+`osc-secnum-fr1`). This certification ensures that the datacenters have implemented a high level of security to protect
```
Comment (leo-scalingo) [inline src/_posts/compliance/security-overview/2000-01-01-compliance.md:32]:
```suggestion
`osc-secnum-fr1`). This qualification ensures that the datacenters have implemented a high level of security to protect
```

File: src/_posts/compliance/security-overview/2000-01-01-compliance.md (hunk 2 above)
Comment (leo-scalingo) [inline src/_posts/compliance/security-overview/2000-01-01-compliance.md:33]:
```suggestion
the data hosted on them. We are working on a SecNumCloud Visa for a new range of products.
```

File: src/_posts/compliance/security-overview/2000-01-01-physical-security.md [hunk 16]
```diff
@@ -0,0 +1,37 @@
...
+title: Physical Security
+nav: Physical
+modified_at: 2024-06-20 00:00:00
+tags: compliance security measures
+index: 1
+---
+
+### Datacenters
+
+Scalingo uses datacenters that are certified ISO 27001, ISO 27017, ISO 27018, HDS (Region `osc-secnum-fr1`) and
```
Comment (leo-scalingo) [inline src/_posts/compliance/security-overview/2000-01-01-physical-security.md:11]:
```suggestion
Scalingo uses datacenters that are certified ISO 27001, ISO 27017, ISO 27018, HDS (Region `osc-fr1`) and
```

I think it was a mistake

File: src/_posts/compliance/security-overview/2000-01-01-physical-security.md [hunk 17]
```diff
@@ -0,0 +1,37 @@
...
+SecNumCloud (Region `osc-secnum-fr1`). These certifications ensure that the datacenters have implemented a high level of
```
Comment (leo-scalingo) [inline src/_posts/compliance/security-overview/2000-01-01-physical-security.md:12]:
```suggestion
and can additionally have the qualification SecNumCloud (Region `osc-secnum-fr1`). These certifications ensure that the datacenters have implemented a high level of
```

File: src/_posts/compliance/security-overview/2000-01-01-physical-security.md (hunk 6 above)
Comment (leo-scalingo) [reply src/_posts/compliance/security-overview/2000-01-01-physical-security.md:14]: We may want to keep these information written down here, but it must be clear that it's related to our N-1 provider no?

File: src/_posts/compliance/2000-01-01-security-overview.md (hunk 13 above)
Comment (leo-scalingo) [reply src/_posts/compliance/2000-01-01-security-overview.md:1]: I don't know what's best it was a bit disturbing, maybe it's not `Security Overview` but `Policies Overview` or `Trust Center` as we discussed it. I don't know actually

File: src/_posts/compliance/security-overview/2000-01-01-compliance.md (hunk 2 above)
Comment (yanjost) [reply src/_posts/compliance/security-overview/2000-01-01-compliance.md]: Not at the moment

File: src/_posts/compliance/security-overview/2000-01-01-physical-security.md (hunk 6 above)
Comment (yanjost) [reply src/_posts/compliance/security-overview/2000-01-01-physical-security.md]: Fixed

**Review by SC-Jerome-S — APPROVED** [review]
Changes accepted

File: src/_posts/compliance/security-overview/2000-01-01-incident-support.md (hunk 4 above)
Comment (yanjost) [reply src/_posts/compliance/security-overview/2000-01-01-incident-support.md]: It's not support availability : it's only the SRE operators

**Pull request merged**